pom.xml

```diff
@@ -55,15 +55,15 @@
         <kotlin.version>1.6.10</kotlin.version>
         <kotlin.coroutines.version>1.6.0</kotlin.coroutines.version>
         <neo4j.version>4.4.3</neo4j.version>
-        <kafka.version>2.4.1</kafka.version>
+        <kafka.version>3.5.1</kafka.version>
         <jackson.version>2.13.1</jackson.version>
         <kotlin.compiler.incremental>true</kotlin.compiler.incremental>
         <neo4j.java.driver.version>4.4.2</neo4j.java.driver.version>
```

Comment:
There's 4.4.12

Reply:
We can skip this is used by the kafka connector which is not used here anymore